application-mocca-calendar-ui/src/main/resources/MoccaCalendar/Macro.xml

```diff
@@ -1212,7 +1229,7 @@ require(['jquery', 'moccaCalendar'], function(jQuery) {
     dayNamesShort: $services.localization.render('xwiki.calendar.dayNamesShort'),
     allDayText: "$escapetool.javascript($services.localization.render('xwiki.calendar.allDayText'))",
     axisFormat: "$escapetool.javascript($services.localization.render('xwiki.calendar.axisFormat'))",
-    timeFormat: "$!escapetool.javascript($services.localization.render('xwiki.calendar.timeFormat'))",
+    timeFormat: "$timeFormat",
```

Comment:
I would still escape the time format. Better safe than sorry (because the date format still comes from the user). Also, if the ``xwiki.calendar.timeFormat`` key is not used anymore then you need to deprecate it. See for instance https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-tree/xwiki-platform-index-tree-macro/src/main/resources/XWiki/DocumentTreeTranslations.xml#L89 .

Reply:
This translation key comes from the macro-fullcalendar app ([here](https://github.com/xwiki-contrib/macro-fullcalendar/blob/master/macro-fullcalendar-ui/src/main/resources/Calendar/Translations.xml#L59)), so I can't mark the key as deprecated here.